pkg/controller/bootstrap/testdata/bootstrap/cluster-config.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+data:
+  install-config: |
+    apiVersion: v1
+    capabilities:
+      baselineCapabilitySet: vCurrent
+    compute:
+    - architecture: amd64
+      hyperthreading: Enabled
+      name: worker
+      platform:
+        azure:
+          osDisk:
+            diskSizeGB: 0
+            diskType: ""
+          osImage:
+            offer: ""
+            plan: ""
+            publisher: ""
+            sku: ""
+            version: ""
+          type: ""
+      replicas: 3
+    controlPlane:
+      architecture: amd64
+      hyperthreading: Enabled
+      name: master
+      platform:
+        azure:
+          osDisk:
+            diskSizeGB: 0
+            diskType: ""
+          osImage:
+            offer: ""
+            plan: ""
+            publisher: ""
+            sku: ""
+            version: ""
+          type: ""
+      replicas: 3
+kind: ConfigMap
```

Comment:
This is the configmap causing the error in the bootstrap_unit test logs that says `bootstrap_test.go:432: Unknown object type *v1.ConfigMap`